src/index.md

```diff
@@ -981,6 +981,77 @@ For the purposes of brevity this document will use the term "bug" to refer to bo
+| `--with-boot-jdk` | Tell configure what boot JDK to use to build the Java libraries. |
+| `--with-debug-level` | Set the debug level. Available levels are `release`, `fastdebug`, `slowdebug`, `optimized`. |
+
+### Working with multiple configurations
+
+Through the configure flags you will select what configuration of the JDK to build. The name of the output directory for the build depends on this configuration. In the example above the JDK ended up in `linux-x86_64-server-release`. This means that we made a release build of a 64 bit linux x86 version of the server JDK. If we change some of these options the output directory will be affected accordingly.
+
+`--with-debug-level` is one example of a configure option that will change the output directory name. Sometimes it makes sense to have several different configurations in parallel. For example while debugging some code you might want to have both a debug build and a release build to be able to test it properly. The directory naming scheme makes this very easy. Simply configure and build the JDKs you need and they will end up next to each other in the build directory.
+
+In the example above we built a `relase` image. To build a debug image as well we can configure with `--with-debug-level=slowdebug`. This will give us a JDK where for instance asserts in the JDK source code are enabled. To select which JDK to work with in later calls to `make` add `CONF=<configuration>`.
+
+    $ sh ./configure --with-boot-jdk=$HOME/jdk-16/ --with-debug-level=slowdebug
+    $ make CONF=slowdebug images
+    $ ls build/
+    linux-x86_64-server-release
+    linux-x86_64-server-slowdebug
+
+## Make targets
+
+`make images`, as used in the example above, will build a JDK image which is very close to what you'd get from any JDK provider. There are several other make targets you can use depending on what you're looking for. The table below contains some commonly used make targets.
+
+| Target | What it does      |
+|:-------|:----------|
+| `exploded-image` | This is the default make target that you'll get if you simply invoke `make`. |
+| `image` | Builds a complete JDK image. A good target to use if you want to build a JDK for general usage or if you want to test something closer to the shipping product. This can also be a good target to use if doing something which might have a build aspect to it. |
+| `<name>-image` | Build just the image for any of jdk, test, docs, symbols, etc. |
+| `reconfigure` | Re-runs the configure script with the same arguments as given the last time. |
+| `demos` | Builds the demos which for instance make it easy to test something UI related. |
+| `docs` | Builds the javadoc. Note that a number of classes in the javadoc API are generated during the build, so `make docs` might do more than simply invoke `javadoc`, depending on the state of your build. |
+| `java.base` | Builds the base module. You can (re)build any module with `make <module>`. |
+| `hotspot` | Builds only the JVM. |
```

Comment:
And any dependencies, which includes quite a bit actually.

Reply:
Rephrased.